src/coreclr/src/System.Private.CoreLib/CreateRuntimeRootILLinkDescriptorFile.targets

```diff
@@ -18,7 +18,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <_ILLinkDescriptorsFilePaths Include="$(MSBuildThisFileDirectory)ILLinkTrim.xml" />
+    <_ILLinkDescriptorsFilePaths Include="$(MSBuildThisFileDirectory)\src\ILLink\ILLinkTrim.xml" />
```

Comment:
MSBuildThisFIleDirectory should already contain `\`

Reply:
Can this use `$(ILLinkDirectory)`?

Reply:
Updated to use `$(ILLinkDirectory)`.